packages/integration-tests/src/test-projects/data-storage-auth-with-triggers-ts/amplify/function.ts

```diff
@@ -1,6 +1,9 @@
-import { defineFunction } from '@aws-amplify/backend';
+import { defineFunction, secret } from '@aws-amplify/backend';
 
 export const myFunc = defineFunction({
   name: 'specialTestFunction',
   entry: './func-src/handler.ts',
+  environment: {
+    AMAZON_SECRET: secret('amazonSecret'),
```

Comment:
name this `TEST_SECRET` or something that makes it clear this is not actually a secret